### PR TITLE
EVAKA-4240: desktop calendar view

### PR DESCRIPTION
**Pull request by Joosakur**
Screenshot 2021-08-31 at 13 20 52

### CODE REVIEW

File: frontend/src/citizen-frontend/calendar/CalendarListView.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export interface Props {
   selectDate: (date: LocalDate) => void
 }
 
-export default React.memo(function CalendarListView({
+export default React.memo(function CalendarGridView({
```
Comment (jaahon) [inline frontend/src/citizen-frontend/calendar/CalendarListView.tsx:21]: You have the two function names mixed up.

File: frontend/src/citizen-frontend/calendar/CalendarGridView.tsx [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    } else {
+      return [
+        ..._.dropRight(weekly),
+        {
+          ...last,
+          dailyReservations: [...last.dailyReservations, daily]
+        }
+      ]
+    }
+  }, [] as WeekProps[])
```
Comment (jaahon) [inline frontend/src/citizen-frontend/calendar/CalendarGridView.tsx:44]: You could just do this higher up the component tree so you wouldn't have to do the same processing twice.

File: frontend/src/citizen-frontend/calendar/CalendarGridView.tsx [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+  justify-content: space-between;
+  margin-bottom: ${defaultMargins.m};
+`
+
+const DayCellDate = styled.div<{ holiday: boolean }>`
+  font-family: Montserrat, sans-serif;
+  font-style: normal;
+  color: ${(p) => (p.holiday ? colors.greyscale.dark : colors.blues.dark)};
+  font-weight: 600;
+  font-size: 20px;
```
Comment (jaahon) [inline frontend/src/citizen-frontend/calendar/CalendarGridView.tsx:148]: Prefer relative units with font-size.

File: frontend/src/citizen-frontend/calendar/CalendarGridView.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+  line-height: 30px;
```
Comment (jaahon) [inline frontend/src/citizen-frontend/calendar/CalendarGridView.tsx:149]: Seems unnecessary.

**Review by jaahon — COMMENTED** [review]
Good job 😄 

I have a couple of comments about the visuals:
* The week number is not lined up with the dates
* There's a double border between all the cells

File: frontend/src/citizen-frontend/calendar/CalendarListView.tsx (hunk 1 above)
Comment (Joosakur) [reply frontend/src/citizen-frontend/calendar/CalendarListView.tsx:21]: oops!

**Comment by Joosakur** [comment]
getting the borders to work was tricky 😅

**Review by jaahon — APPROVED** [review]
Beautiful 💎

**Pull request merged**